api/v1beta1/common.go

```diff
@@ -78,31 +141,30 @@ type NodeSection struct {
 
 // DeployStrategySection for fields controlling the deployment
 type DeployStrategySection struct {
-
-	// +kubebuilder:default=true
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:booleanSwitch"}
 	// Deploy boolean to trigger ansible execution
+	// +kubebuilder:default=true
 	Deploy bool `json:"deploy"`
 
-	// +kubebuilder:validation:Optional
 	// AnsibleTags for ansible execution
+	// +kubebuilder:validation:Optional
 	AnsibleTags string `json:"ansibleTags,omitempty"`
 
-	// +kubebuilder:validation:Optional
 	// AnsibleLimit for ansible execution
+	// +kubebuilder:validation:Optional
 	AnsibleLimit string `json:"ansibleLimit,omitempty"`
 
-	// +kubebuilder:validation:Optional
 	// AnsibleSkipTags for ansible execution
+	// +kubebuilder:validation:Optional
 	AnsibleSkipTags string `json:"ansibleSkipTags,omitempty"`
```

Comment:
I'm trying to leave the DeployStrategy implementation untouched atm to avoid any additional scope creep that might come from moving these. But we might be able to consolidate these under that struct as well. Could we possibly investigate that under a seperate PR?